utils/email/index.js

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import fs from 'fs';
+import sendgridEmail from '@sendgrid/mail';
+import mjml2html from "mjml";
+
+if (process.env.SENDGRID_API_KEY) {
+  sendgridEmail.setApiKey(process.env.SENDGRID_API_KEY);
+}
+
+
+function sendEmail({recipient, subject, htmlBody, textBody}) {
+  if (!process.env.SENDGRID_API_KEY) {
+    console.error('SendGrid error: SENDGRID_API_KEY is not set');
+    return;
+  }
+  const from = 'mentors@tupu.io'; // Change to your verified sender
+  recipient = process.env.DEV_EMAIL_RECIPIENT || recipient;
+  subject = buildSubjectLine(subject);
+  const msg = {
+    to: recipient,
+    from: process.env.DEV_EMAIL_RECIPIENT || from, 
+    subject: subject,
+    text: textBody,
+    html: htmlBody,
+  };
+  sendgridEmail
+    .send(msg)
+    .then(() => {
+      console.log(`Email sent to SendGrid for <${recipient}>,\n subject "${subject}"`);
+    })
+    .catch((error) => {
+      console.error('SendGrid error:', error)
+      console.error(`Could not deliver email to <${recipient}>,\n subject "${subject}"`);
```

Comment:
Note to self: we need to set up alerts for this.

Reply:
let me know if I can help with this, it's interesting